addons/sms/static/src/js/fields_sms_widget.js

```diff
@@ -164,6 +164,20 @@ var SmsWidget = FieldTextEmojis.extend({
         this._super.apply(this, arguments);
         this._updateSMSInfo();
     },
+
+    /**
+     * @override
+     * @private
+     */
+    _onBlur: function () {
+        var content = this._getValue();
+        var count = content.length;
+        if( !content.trim().length && count > 0) {
+            this.do_warn(_t("Your SMS Text Message must include at least one non-whitespace character"));
```

Comment:
If you do this in 14 you have to export translations in order to add the string to translate. You can do it in technical, translations, export SMS content in English as .pot file and get only this new sentence (you may have other changes that we will not keep, only the new sentence at its place in the pot file).